src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,18 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+    private User user;
+
     public boolean login(String email, String password) {
+        user = userService.findByEmail(email);
+
+        if (user == null) {
+            return false;
+        } else if (user.getPassword() == password && user.getEmail() == email) {
+            return true;
+        }
+
         return false;
```

Comment:
Let's remove redundant if-else:
```suggestion
        return user != null && user.getPassword().equals(password);
```